provider/provider.go

```diff
@@ -62,6 +62,8 @@ func DatabricksProvider() *schema.Provider {
 			"databricks_mws_networks":               mws.ResourceNetwork(),
 			"databricks_mws_storage_configurations": mws.ResourceStorageConfiguration(),
 			"databricks_mws_workspaces":             mws.ResourceWorkspace(),
+			"databricks_mws_vpc_endpoint":           mws.ResourceVPCEndpoint(),
+			"databricks_databricks_mws_pas":         mws.ResourcePAS(),
```

Comment:
rename to `databricks_mws_private_access_settings`

Reply:
Done